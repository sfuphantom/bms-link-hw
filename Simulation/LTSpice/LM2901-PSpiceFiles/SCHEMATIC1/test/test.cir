** Profile: "SCHEMATIC1-test"  [ C:\Modeling\LM2901_ti\lm2901-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../lm2901.lib" 
* From [PSPICE NETLIST] section of C:\SPB_DATA\cdssetup\OrCAD_PSpice\17.4.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 30u 0 
.OPTIONS ADVCONV
.OPTIONS FILEMODELSEARCH
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
