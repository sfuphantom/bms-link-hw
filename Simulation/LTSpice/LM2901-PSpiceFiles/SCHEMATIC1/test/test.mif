 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=2038, size=965
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=4331, size=750
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=4175, size=155
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=3608, size=566
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=3487, size=120
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=3371, size=115
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=3004, size=366
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=5621, size=349
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=7356, size=166
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=5082, size=178
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=6328, size=120
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=5261, size=178
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=6207, size=120
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=5971, size=235
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=5440, size=180
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=6449, size=164
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=7142, size=213
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=6997, size=144
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=6729, size=125
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=6614, size=114
 lib=C:\Modeling\LM2901_ti\lm2901.lib, offset=6855, size=141
