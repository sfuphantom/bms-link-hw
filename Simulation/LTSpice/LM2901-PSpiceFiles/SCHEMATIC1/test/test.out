
**** 01/31/23 13:54:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Modeling\LM2901_ti\lm2901-pspicefiles\schematic1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../lm2901.lib" 
* From [PSPICE NETLIST] section of C:\SPB_DATA\cdssetup\OrCAD_PSpice\17.4.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 30u 0 
.OPTIONS ADVCONV
.OPTIONS FILEMODELSEARCH
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source LM2901
.EXTERNAL INPUT GND
.EXTERNAL INPUT IN-
.EXTERNAL INPUT Vcc
.EXTERNAL INPUT OUT
.EXTERNAL INPUT IN+
R_R3         VCC OUT  10k TC=0,0 
V_V1         IN+ 0  
+SIN 5 100m 100k 0 0 0
V_VINN         IN- 0 5
V_VINN2         VCC 0 12
R_R2         0 GND  1m TC=0,0 
X_U1         IN+ IN- VCC GND OUT LM2901

**** RESUMING test.cir ****
.END

WARNING(ORPSIM-15223): Library file C:\Modeling\LM2901_ti\lm2901.lib has changed since index file lm2901.ind was created.

WARNING(ORPSIM-15227): The timestamp changed from Mon Jun 27 10:51:40 2022 to Tue Jan 31 10:49:11 2023.

INFO(ORPSIM-15422): Making new index file lm2901.ind for library file lm2901.lib.

Index has 21 entries from 1 file(s).

**** 01/31/23 13:54:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Modeling\LM2901_ti\lm2901-pspicefiles\schematic1\test.sim ] 


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U1.X_DESD3.model22 
          IS   10.000000E-15 
          BV  100            
          RS    5            


               X_U1.X_DESD5.model22 
          IS   10.000000E-15 
          BV  100            
          RS    5            


**** 01/31/23 13:54:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Modeling\LM2901_ti\lm2901-pspicefiles\schematic1\test.sim ] 


 ****     Voltage Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_U1.X_U5.X_SVOL._SVOL 
         RON    1            
        ROFF    1.000000E+12 
         VON    0            
        VOFF    1            


               X_U1.X_U5.X_SMID._SMID 
         RON    1            
        ROFF    1.000000E+12 
         VON    0            
        VOFF    1            


               X_U1.X_U5.X_SHIZ._SHIZ 
         RON    1            
        ROFF    1.000000E+12 
         VON    0            
        VOFF    1            


**** 01/31/23 13:54:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Modeling\LM2901_ti\lm2901-pspicefiles\schematic1\test.sim ] 


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  GND) 275.0E-09  (  IN+)    5.0000  (  IN-)    5.0000  (  OUT)   12.0000      

(  VCC)   12.0000  (X_U1.IN+BUFF)    5.0000                                     

(X_U1.IN-BUFF)    5.0000              (X_U1.N859943)   12.0000                  

(X_U1.N859997)    0.0000              (X_U1.N860177)    5.0004                  

(X_U1.N860391)   11.8810              (X_U1.N860397)    5.9406                  

(X_U1.N860405)   11.8810              (X_U1.N860457) 275.0E-09                  

(X_U1.N860477)    0.0000              (X_U1.X_U3.VH)    0.0000                  

(X_U1.X_U5.1V)    1.0000              (X_U1.X_U7.1V)    1.0000                  

(X_U1.X_U3.OUT)   12.0000             (X_U1.X_U5.MID)    6.0000                 

(X_U1.V+_BUFFER)   12.0000            (X_U1.V-_BUFFER) 275.0E-09                

(X_U1.X_U3.VMID)    6.0000            (X_U1.X_U2.N20155)   10.0000              

(X_U1.X_U2.N20415)    -.1100          (X_U1.X_U2.N20531) 275.0E-09              

(X_U1.X_U2.N20539)    -.1100          (X_U1.X_U2.N20826) 275.0E-09              

(X_U1.X_U2.N20833) 275.0E-09          (X_U1.X_U2.N20840) 275.0E-09              

(X_U1.X_U3.INNNEW)    5.0000          (X_U1.X_U4.N03175)    5.9406              

(X_U1.X_U5.X_U3.3)   12.0000          (X_U1.X_U5.X_U8.6)    0.0000              

(X_U1.X_U2.N202710)   10.0000         (X_U1.X_U2.X_U24.6) 275.0E-09             

(X_U1.X_U5.N774212)   12.0000         (X_U1.X_U5.N774290)   11.0000             

(X_U1.X_U5.N778484)   12.0000         (X_U1.X_U5.N778496)   12.0000             

(X_U1.X_U5.N789513)    1.0000         (X_U1.X_U5.N850209)   12.0000             

(X_U1.X_U5.X_U10.4)    1.0000         (X_U1.X_U7.N779976)   36.0100             

(X_U1.X_U7.N780066)   12.0000         (X_U1.X_U7.N780086)    0.0000             

(X_U1.X_U7.N780252)    1.9900         (X_U1.X_U7.X_U16.4)    0.0000             

(X_U1.X_U2.X_U1.OUT2) 275.0E-09       (X_U1.X_U5.X_U7.VMID)    6.0000           

(X_U1.X_U7.X_U5.OUT2)    0.0000       (X_U1.X_U2.X_U21.OUT2) 275.0E-09          

(X_U1.X_U2.X_U22.OUT2) 275.0E-09      (X_U1.X_U2.X_U23.OUT2) 275.0E-09          

(X_U1.X_U7.X_U13.OUT1)   12.0000      (X_U1.X_U7.X_U15.OUT2)    0.0000          

(X_U1.X_U5.CONTROL_HIZ)    0.0000     (X_U1.X_U5.CONTROL_MID)    1.0000     




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V1         3.490E-09
    V_VINN       3.490E-09
    V_VINN2     -2.750E-04
    X_U1.V_VOS   0.000E+00
    X_U1.V_VHYST   0.000E+00
    X_U1.X_U5.V_VLOGIC   0.000E+00
    X_U1.X_U5.V_V1  -1.001E-09
    X_U1.X_U2.V_VCMNN   0.000E+00
    X_U1.X_U2.V_VCMPN   0.000E+00
    X_U1.X_U2.V_VCMNP   0.000E+00
    X_U1.X_U2.V_VCMPP   0.000E+00
    X_U1.X_U7.V_VS_MIN_SET   0.000E+00
    X_U1.X_U7.V_VS_MAX_SET   0.000E+00
    X_U1.X_U7.V_VLOGIC   0.000E+00

    TOTAL POWER DISSIPATION   3.30E-03  WATTS



          JOB CONCLUDED

**** 01/31/23 13:54:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Modeling\LM2901_ti\lm2901-pspicefiles\schematic1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .80
